.github/workflows/scripts/load-oci-archives.sh

```diff
@@ -51,14 +51,37 @@ echo "Importing ${OCI_IMAGES[*]} into docker".
 for img in "${OCI_IMAGES[@]}"; do
     oci_dir="ocidir://${ROOTDIR}oci/${img}"
     platform_tar="${img}-${PLATFORM}-image.tar"
-    
+
     # regclient works with directories rather than tars, so import the OCI tar to a directory
     regctl image import "$oci_dir" "${img}-image.tar"
     dig="$(regctl image digest --platform "$PLATFORM" "$oci_dir")"
     # export the single platform image using the digest
     regctl image export "$oci_dir@${dig}" "${platform_tar}"
-    
-    docker load < "${platform_tar}"
-    docker image tag "localhost/oci/${img}:latest" "${img}:latest-local"
-    docker image rm "localhost/oci/${img}:latest"
+
+    # Load the image and capture the output
+    load_output=$(docker load < "${platform_tar}")
+    echo "$load_output"
+
+    # Extract the image reference from the output. Docker load can output either:
+    # - "Loaded image ID: sha256:..." (use the sha256)
+    # - "Loaded image: <name>" or "Loaded image: <name>@sha256:..." (use the full name)
+    # We try to extract the reference from either format
+    image_ref=""
+
+    # Try "Loaded image ID: sha256:..." format first
+    image_ref=$(echo "$load_output" | grep "Loaded image ID:" | awk '{print $4}')
```

Comment:
I think this might be the same as `dig` above. On my machine I get this:
```
$regctl image digest ocidir:///tmp/spire-server
sha256:8f953b542dc6388db74453c4a84d70568626d88729444b456dcb3d6e40923bc2
```